### ISSUE TITLE
v1.21.1 is 32 bit, won't run on MacOS 10.15 Catalina

### ISSUE THREAD

**Issue by tomhundt** [nightingale-media-player/nightingale-hacking#386]
I'm running a beta of MacOS 10.15 Catalina on my Mac; it's refusing to
run Nightingale, saying it's 32 bit.  (Appears in Applications with icon
greyed out and slashed through.)  Only that version is available from [getnightingale](http://getnightingale.com) download site.  

You can confirm 32-bitness if you have it
installed on Mojave 10.14.6, by doing Apple > About this Mac > System
Report > Software > Applications, where you see a list of what's
installed and the column "64-Bit (Intel)". There are other ways to get
the same information, e.g., by running a program called Go64.

**Comment by freaktechnik** [comment]
Yep. Given that the last commit in this repo was 4 years ago, I doubt a 64 bit version will just appear (plus XULRunner isn't the platform anymore that it used to be even 4 years ago...). Sorry.

**Comment by drkvogel** [comment]
I just discovered my new favourite music player, Nightingale, and now I hear it's going to stop working after I upgrade to MacOS Catalina later today... Does anyone think it's worth reviving this project and getting it to work on Calalina, or is there another open source music player (that works on Mac) that I could use and maybe contribute to?

**Comment by AntoineTurmel** [comment]
If I'm not wrong x64 builds of Firefox for Mac OS X started with Firefox 4...
The current Nightingale engine is based on Firefox 3.6.x, we don't have any deps/target for Mac OS X 64 bits.
We started to upgrade our XULRunner engine, but eventually the project stalled... 
I wonder how many apps will broke with Catalina due to no 64 bits support...

@drkvogel I used musikCube which is a nice terminal player : https://github.com/clangen/musikcube

**Comment by darealshinji** [comment]
macOS Catalina doesn't start 32 bit apps anymore?

**Comment by AntoineTurmel** [comment]
> 
> 
> macOS Catalina doesn't start 32 bit apps anymore?

Unfortunately no... see https://en.wikipedia.org/wiki/MacOS_Catalina#Removed_or_changed_components

**Comment by AntoineTurmel** [comment]
One workaround could be running Nightingale Windows build with Wine on macOS ?

**Comment by darealshinji** [comment]
>> macOS Catalina doesn't start 32 bit apps anymore?

> Unfortunately no... see https://en.wikipedia.org/wiki/MacOS_Catalina#Removed_or_changed_components

Wow, bad decision. I hope that won't happen to Ubuntu too soon. Using wine may work but seems strange if there's a native build that technically works. Another alternative would be to look for a different player that feels similar to Nightingale.

**Comment by AntoineTurmel** [comment]
Looks like 32 bits apps won't run neither using Wine 😞 https://www.pcmag.com/article/371267/how-to-run-32-bit-apps-in-macos-catalina

**Comment by AntoineTurmel** [comment]
Ok, one possible workaround:
Using : https://github.com/linux-noah/noah with Linux x64 bit build:
https://github.com/nightingale-media-player/nightingale-hacking/releases/download/nightingale-1.12.1/Nightingale_1.12.1-2454_linux-x86_64.tar.bz2

(You will need Xserver also)
